README.rst

```diff
@@ -119,42 +119,43 @@ Install ``pyglmnet`` using ``setup.py`` as follows
 Getting Started
 ~~~~~~~~~~~~~~~
 
+
 Here is an example on how to use the ``GLM`` estimator.
+.. This example is also found in examples/intro_example.py.
```

Comment:
@tommyod, it passed my attention when reviewing. Looks like you forgot to commit this file